### PR TITLE
Add documentation for ActionCodeURL

### PR DESCRIPTION
**Pull request by avolkovi**

### CODE REVIEW

File: docs-exp/auth.actioncodeurl.operation.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 ## ActionCodeURL.operation property
 
+The action performed by the email action link. It returns from one of the types from
```
Comment (avolkovi) [inline docs-exp/auth.actioncodeurl.operation.md:7]: looks like the @link got truncated

File: docs-exp/auth-types.actioncodeurl.operation.md [hunk 2]
```diff
@@ -4,6 +4,8 @@
 
 ## ActionCodeURL.operation property
 
+The action performed by the email action link. It returns from one of the types from [ActionCodeInfo](./auth-types.actioncodeinfo.md)
```
Comment (samhorlbeck) [inline docs-exp/auth-types.actioncodeurl.operation.md:7]: does {@link ActionCodeInfo} work?

File: docs-exp/auth.actioncodeurl.operation.md (hunk 1 above)
Comment (samhorlbeck) [reply docs-exp/auth.actioncodeurl.operation.md:7]: ah that answers it

File: docs-exp/auth-types.actioncodeurl.operation.md (hunk 2 above)
Comment (avolkovi) [reply docs-exp/auth-types.actioncodeurl.operation.md:7]: looks like it works in the auth-types docs but not the auth docs, probably will self resolve once I add documentation for ActionCodeInfo in both

File: packages-exp/auth-exp/src/core/action_code_url.ts [hunk 3]
```diff
@@ -45,11 +49,20 @@ const MODE_TO_OPERATION_MAP: { [key: string]: externs.Operation } = {
 
 /**
  * Maps the mode string in action code URL to Action Code Info operation.
+ *
+ * @param mode
+ * @internal
```
Comment (samhorlbeck) [inline packages-exp/auth-exp/src/core/action_code_url.ts:54]: You don't need to do this right? Since these are not exported?

**Review by samhorlbeck — APPROVED** [review]

File: packages-exp/auth-exp/src/core/action_code_url.ts (hunk 3 above)
Comment (avolkovi) [reply packages-exp/auth-exp/src/core/action_code_url.ts:54]: I think you're right, doing it for consistency here anyway, no promises I'll do it everywhere

**Review by Feiyang1 — APPROVED** [review]

**Pull request merged**